Makefile is broken on osx with zsh

**Expected behavior**:
For `make build` on edge branch to build successfully

**Actual behavior**:
I get this error:

```
$ make build
ENV_FILE=.env
POSTGRES_DOCKER= true
TAG= dev
docker compose -f docker-compose.yml -f docker-compose.dev.yml --profile postgres --env-file .env build
[+] Building 0.0s (0/0)                                                                                                                                                                       docker:colima
invalid tag "docker.io/compdem/polis-postgres:\tdev": invalid reference format
make: *** [build] Error 17
```

**To Reproduce**:
Git clone and run `make build` or `make echo_vars`

Note the tabs "\t" in the output, and the whitespace in the vars

**Screenshots**:
<img width="757" alt="Screenshot 2024-10-17 at 2 18 20 PM" src="https://github.com/user-attachments/assets/7e51098d-8356-48d6-ba1a-1e3815540322">

**Device information**:
Current environment:
- OSX Sonoma 14.5
- zsh 5.9 (x86_64-apple-darwin23.0)
- **colima** version 0.7.5 (alternative docker-compat container runtime)
  - this is the only "weird" thing, but I isolated a fix in the makefile environmen
- Docker version 27.2.1, build 9e34c9bb39
- Docker Compose version 2.29.2
- GNU Make 3.81

**Additional context**:
None

## Solution

I've found a fix that I will share shortly. Basically, issue involves make format, the `define` method, and whitespace in a place that make doesn't like it :)

--- a/Makefile
+++ b/Makefile
@@ -9,11 +9,11 @@ SHELL=/bin/bash
 E2E_RUN = cd e2e;
 
 define parse_env_value
-	$(shell grep -e ^$(1) ${ENV_FILE} | awk -F'[=]' '{gsub(/ /,"");print $$2}')
+$(shell grep -e ^$(1) ${ENV_FILE} | awk -F'[=]' '{gsub(/ /,"");print $$2}')
 endef
 
 define parse_env_bool
-  $(if $(filter false,$(1)),false,true)
+$(if $(filter false,$(1)),false,true)
 endef
 
 # Default environment and settings (dev)

